blocks/log/json.py

```diff
@@ -0,0 +1,54 @@
+from mimir import Logger
+
+from .log import TrainingLogBase
+
+
+class PiclableLogger(object):
+    def __init__(self, filename, **kwargs):
+        logger = Logger(**kwargs)
+        self.__dict__ = logger.__dict__
```

Comment:
Is it ugly? I don't know what is the right way to inherit from the objects created with a factory.